controller/persistence/config_store.go

```diff
@@ -62,6 +63,10 @@ func (entity *Config) SetValues(ctx *boltz.PersistContext) {
 	ctx.SetString(FieldName, entity.Name)
 	ctx.SetString(FieldConfigType, entity.Type)
 	ctx.SetMap(FieldConfigData, entity.Data)
+
+	if entity.Data == nil {
+		ctx.Bucket.SetError(errorz.NewFieldError("data is required", "data", nil))
```

Comment:
is this accurate? wasn't @gberl002 sending data just that data didn't deserialize properly due to the format? would you consider adding "or data improperly formatted" to this response? (assuming I understand what happened)

Reply:
Not at this level.

Reply:
not at this level 'in the code' - but somewhere else? I just want the consumer to get a good message to help them understand the problem. As long as a consumer gets a "hey you screwed up" (speaking of which is this now a 400) and 'how' that's all I care deeply about.

Reply:
https://github.com/openziti/edge/pull/1021/files#diff-aabaaa26930c9df6015bb4c88a9ceae28fafc4e3431545f3c916202c630648cdR50